index.md

```diff
@@ -4,15 +4,6 @@ layout: lesson
 This Library Carpentry lesson introduces people working in library- and information-related roles to working with data in OpenRefine. At the conclusion of the lesson you will understand what the OpenRefine software does and how to use the OpenRefine software to work with data files.
 
 > ## Prerequisites
-> To complete this lesson you will need to install [OpenRefine](http://openrefine.org/download.html) and download the file [doaj-article-sample.csv](https://github.com/LibraryCarpentry/lc-open-refine/raw/gh-pages/data/doaj-article-sample.csv).
->
-> OpenRefine requires one of these web browsers:
-> * Google Chrome
-> * Chromium
-> * Opera
-> * Microsoft Edge
-> 
-> OpenRefine has some issues with Firefox. Internet Explorer is not supported. 
-> 
+> To complete this lesson you will need to install OpenRefine and download a data file.
```

Comment:
I think it makes sense to keep the download links here in the text. Happy to discuss further the pros and cons but it's not a factor in the OpenRefine version so think we could deal with that discussion separately

Reply:
We might need to check with Brynne, the new Accessibility Coordinator.  If the link appears several times on the same page, screenreaders become painful.  Maybe refer to the Setup page?

Reply:
Let's move this part into https://github.com/LibraryCarpentry/lc-open-refine/pull/248